### PR TITLE
Use single timeout window in e2e tests

### PR DESCRIPTION
**Pull request by nikhita**
Uses a single timeout window of 10 minute for e2e tests, but keeps the granual feedback added in https://github.com/kubermatic/kubermatic/pull/4348.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: api/cmd/conformance-tests/runner.go [hunk 1]
```diff
@@ -343,38 +343,43 @@ func (r *testRunner) executeScenario(log *zap.SugaredLogger, scenario testScenar
...
 		"[Kubermatic] Wait for machines to get a node",
 		report,
 		func() error {
-			return waitForMachinesToJoinCluster(log, userClusterClient, nodeJoinTimeout)
+			return waitForMachinesToJoinCluster(log, userClusterClient, overallTimeout)
 		},
 	); err != nil {
 		return report, fmt.Errorf("failed to wait for machines to get a node: %v", err)
 	}
+	timeoutLeft := time.Duration(overallTimeout.Minutes() - time.Since(startTime).Minutes())
```
Comment (alvaroaleman) [inline api/cmd/conformance-tests/runner.go]: I _think_ the issue is that `time.Duration` is an int64 representing nanoseconds. What you do here is taking the number of minutes left and interpreting them as nanoseconds.

You could probably factor this out into a function in order to be able to write a unittest for it.

**Comment by nikhita** [comment]
Tests passing and ready for review. :tada:

**Pull request merged**